python/ray/serialization.py

```diff
@@ -121,6 +141,20 @@ def object_ref_reducer(obj):
         self._register_cloudpickle_reducer(ray.ObjectRef, object_ref_reducer)
         serialization_addons.apply(self)
 
+        # None means match all strings
+        self._pickle_whitelist = None
+        whitelist_path = ray_constants.RAY_WHITELIST_PATH
```

Comment:
Can we put the logic about whitelist in a separate file? We should try to avoid code conflict.

Reply:
ok, I rename it to RAY_PICKLE_WHITELIST_CONFIG_PATH